CHANGELOG.md

```diff
@@ -7,178 +7,185 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.36] - 2024-09-24
+
+### Added
+
+- Added logic to CronJob Update to set `isAnyHostAllowed`
```

Comment:
Isn't this a `fix` versus a new feature?

Reply:
fixed